.github/workflows/spelling.yml

```diff
@@ -0,0 +1,28 @@
+name: Spell checking
+on: [push, pull_request]
+
+jobs:
+  spelling:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout-merge
```

Comment:
Any idea why PRs and push events are separate? I'd just do:
```yml
- uses: actions/checkout@v2
```
but maybe I'm missing something.

Reply:
It was given in the template already and I did not tamper with it. Plus, I don't have any idea regarding this too 😅

Reply:
I will remove this as well and lets see if it causes some sort of error.